src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -4615,14 +4620,13 @@ checkNullable:
                 value = ParseExpressionCore()
 
             ElseIf modifiers.Any AndAlso modifiers.Any(SyntaxKind.OptionalKeyword) Then
-
-                equals = ReportSyntaxError(InternalSyntaxFactory.MissingPunctuation(SyntaxKind.EqualsToken), ERRID.ERR_ObsoleteOptionalWithoutValue)
-                value = ParseExpressionCore()
+                'TODO:           Make this dependant on language feature flag
+                'equals = ReportSyntaxError(InternalSyntaxFactory.MissingPunctuation(SyntaxKind.EqualsToken), ERRID.ERR_ObsoleteOptionalWithoutValue)
+                'value = ParseExpressionCore()
 
             End If
 
             Dim initializer As EqualsValueSyntax = Nothing
-
             If value IsNot Nothing Then
```

Comment:
Please keep an empty line before the ```If```. In general, please refrain from making reformatting/restyling existing code, and, if you absolutely must, please do not mix it with a feature work. This would make PR process much smother.